build.gradle

```diff
@@ -114,6 +114,7 @@ dependencies {
     compile 'com.google.inject.extensions:guice-servlet:4.0'
     compile 'org.codehaus.jackson:jackson-core-asl:1.9.13'
     compile 'org.codehaus.jackson:jackson-mapper-asl:1.9.13'
+    compile 'org.codehaus.jackson:jackson-jaxrs:1.9.13'
```

Comment:
I'm not familiar with JSON library but is jackson-jaxrs necessary?

Reply:
Sorry, it's my mistake during trial. Let me remove this library.